bootstrap/config/gcp_prototype.yaml

```diff
@@ -0,0 +1,44 @@
+# Sample config for kubeflow bootstrapper
+---
+# Apps that always apply
+DefaultApp:
```

Comment:
Why do the key names start with capital letters? This doesn't follow standard K8s or GCP deployment manager syntax. Can we make it lower case?